Source/BestProjectEver/Private/Character/BPE_Enemy.cpp

```diff
@@ -180,14 +183,14 @@ FRotator ABPE_Enemy::GetViewRotation() const
 }
 
 //----------------------------------------------------------------------------------------------------------------------
-void ABPE_Enemy::SetColorType(const EColorType NewColorType)
+void ABPE_Enemy::SetDefaultColorType(const EColorType NewColorType)
 {
-	ColorType = NewColorType;
-	IBPE_InteractWithColorType::SetColorType(NewColorType);
+	DefaultColorType = NewColorType;
+	IBPE_InteractWithColorType::SetDefaultColorType(NewColorType);
 }
 
 //----------------------------------------------------------------------------------------------------------------------
-void ABPE_Enemy::UpdateMeshColor()
+void ABPE_Enemy::UpdateMeshColorType(const EColorType ColorType)
```

Comment:
Should this be called UpdateMeshColorType? When its purpose it's not to only update the mesh?

Reply:
![image](https://user-images.githubusercontent.com/85327912/176215238-097cbf2e-954d-4789-b943-521327b0f9f2.png)